DQM/HLTEvF/python/HLTObjectsMonitor_B2G_cfi.py

```diff
@@ -0,0 +1,83 @@
+import FWCore.ParameterSet.Config as cms
+
+b2gObjects = cms.VPSet(
+       cms.PSet(
```

Comment:
Hi @mtosi - instead of having many copies of what is essentially the same PSet, please make a cfi with just the PSet and then import and copy it for each producer with the small needed changes)